src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -460,8 +462,12 @@ internal SqlStatistics Statistics
             {
                 if (null != _activeConnection)
                 {
-                    if (_activeConnection.StatisticsEnabled ||
-                        _diagnosticListener.IsEnabled(SqlClientDiagnosticListenerExtensions.SqlAfterExecuteCommand))
+                    if (_activeConnection.StatisticsEnabled
+#if netcoreapp
+                        || _diagnosticListener.IsEnabled(SqlClientDiagnosticListenerExtensions.SqlAfterExecuteCommand))
+#else
+                        )
+#endif
```

Comment:
The compiler directives feel bad when there's a netcoreapp version of this file already split out in the PR. Just checking that this is because it's prohibitively difficult to do this any other way, if you've deliberately chosen to use both then it's ok.

Reply:
I just updated the code to work with platform specific implementations instead.